### PR TITLE
Added `tuning_method` & `seed` attributes and updated documentation

### PR DESCRIPTION
**Pull request by Advitya17**
`tuning_method`: This brings an effective Hyperparameter Tuning strategy to a declarative level where the user can specify (should they wish to tune hyper parameters for their downstream model) 'TrainValidationSplit' or 'CrossValidator' as their tuning strategy. Combining this with `extra_config`, the user is expected to have nearly total control in the parameter tuning process, by use their preferred tuning strategy with the grid of their choice. ('TrainValidationSplit' may be more recommended for large datasets though.)

`seed`: This brings the random seeding of user's workload to a declarative level as well, where the user may specify a particular seed to be used in the entire model selection (and hyperparameter tuning, if applicable) process. This will enable result reproducibility, an important component in Machine Learning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,24 +44,28 @@ Materialization Trade-offs for Feature Transfer from Deep CNNs for Multimodal Da
...
+     * dS            : number of structured features
+     * model_name    : Name of the (PySpark MLLib) Downstream ML Model to run in the Vista optimizer
+     * extra_config  : Extra configuration settings for hyperparameter tuning with the downstream model
+     * tuning_method : Method (TrainValidationSplit / CrossValidator) to use for hyperparameter tuning.
+     * seed          : Random Seed to set for all data split / algorithm training tasks for reproducibility in result. 
     **/
     vista = Vista("vista-example", 32, 8, 8, 'alexnet', 4, 0, 'hdfs://../foods.csv',
-                      'hdfs://.../images', 20129, 130, model_name='LogisticRegression', extra_config={})
+                      'hdfs://.../images', 20129, 130, model_name='LogisticRegression', extra_config={}, 
+                      tuning_method=None, seed=2019)
```
Comment (scnakandala) [inline README.md:65]: What is the default for tuning_method? Put that value here.

Comment (Advitya17) [reply README.md:65]: `None`, already done and discussed.

**Comment by Advitya17** [comment]
Also added `test_size` attribute to the Vista API as a commit right now. Let me know if you can see the commit.

**Comment by Advitya17** [comment]
@scnakandala I think we forgot to merge this lol

**Pull request merged**